src/plugins/intel_gpu/src/graph/concatenation.cpp

```diff
@@ -69,7 +69,6 @@ std::vector<layout> concatenation_inst::calc_output_layouts(const concatenation_
     }
     ov::op::v0::Concat op;
     op.set_friendly_name(desc->id);
-    op.set_concatenation_axis(axis_index);
```

Comment:
Why did you remove that? I think it should be replaced with `set_axis()` instead

Reply:
Done